bql/semantic/expression_test.go

```diff
@@ -189,22 +190,30 @@ func TestBooleanEvaluationNode(t *testing.T) {
 	}
 }
 
-func mustBuildLiteral(textLiteral string) *literal.Literal {
+func mustBuildLiteral(t *testing.T, textLiteral string) *literal.Literal {
 	lit, err := literal.DefaultBuilder().Parse(textLiteral)
 	if err != nil {
-		panic(fmt.Sprintf("could not parse text literal %q, got error: %v", textLiteral, err))
+		t.Fatalf("could not parse text literal %q, got error: %v", textLiteral, err)
 	}
 	return lit
 }
 
-func mustBuildNodeFromStrings(nodeType, nodeID string) *node.Node {
+func mustBuildNodeFromStrings(t *testing.T, nodeType, nodeID string) *node.Node {
 	n, err := node.NewNodeFromStrings(nodeType, nodeID)
 	if err != nil {
-		panic(fmt.Sprintf("could not build node from type %q and ID %q, got error: %v", nodeType, nodeID, err))
+		t.Fatalf("could not build node from type %q and ID %q, got error: %v", nodeType, nodeID, err)
 	}
 	return n
 }
 
+func mustBuildTime(t *testing.T, timeLiteral string) *time.Time {
```

Comment:
Functions that perform set up or teardown tasks such as constructing input messages are usually known as "Test helpers"

Whenever you pass a *testing.T, its' recommended to call t.Helper() to attribute failures in the test helper to the line where helper is called.

https://golang.org/pkg/testing/#B.Helper

Reply:
It is pretty nice that we have this in Go. Thank you for the comment Jose, I have just done so. 😀